articles/synapse-analytics/sql-data-warehouse/maintenance-scheduling.md

```diff
@@ -57,6 +57,13 @@ To view the maintenance schedule that has been applied to your Synapse SQL pool,
 
 ![Overview blade](./media/maintenance-scheduling/clear-overview-blade.PNG)
 
+## Skip or disable maintenance schedule
+
+To ensure compliance with latest security requirements, we are unable to accommodate requests to skip or delay these updates. However, you may have some options to adjust your maintenance window within the current cycle depending on your situation:
+- If you receive a pending notification for maintenance, and you need more time to finish your jobs or notify your team, you can change the window start time as long as you do so before the beginning of your defined maintenance window. This will shift your window forward in time within the cycle. Note that if you change the window to a start time before the actual present time, maintenance will be triggered immediately.
+- You can manually trigger the maintenance by pausing and resuming (or scaling) your SQL Dedicated pool after the start of a cycle for which a "Pending" notification has been received. The weekend maintenance cycle starts on Saturday at 00:00 UTC; the midweek maintenance cycle starts Tuesday at 12:00 UTC.
+- Although we do require a minimum window of 3 hours, note that maintenance usually takes less than 30 minutes to complete, but it may take longer in some cases. For example, if there are active transactions when the maintenance starts, they will be aborted and rolled back, which may cause delays in coming back online. To avoid this scenario, we recommend that you ensure that no long-running transactions are active during the start of your maintenance window.
+
 ## Change a maintenance schedule
 
 A maintenance schedule can be updated or changed at any time. If the selected instance is going through an active maintenance cycle, the settings will be saved. They'll become active during the next identified maintenance period. [Learn more](../../service-health/resource-health-overview.md?toc=/azure/synapse-analytics/sql-data-warehouse/toc.json&bc=/azure/synapse-analytics/sql-data-warehouse/breadcrumb/toc.json) about monitoring your data warehouse during an active maintenance event.
```

Comment:
This is not strictly correct -- and somewhat contradicts the options provided above.  Not sure whether it is better to merge the "Skip or disable maintenance schedule" together with this section, i.e. have one section which is "Change a maintenance schedule", or just refer to the instructions in the previous section.  Or just clarify -- instead of "They'll become active during the next identified maintenance period." to something like "If the selected instance is going through an active maintenance cycle, you can change the window start time as long as you do so before the beginning of your defined maintenance window.  See ##Skip or disable maintenance schedule for "

Reply:
Thanks Jim. so maybe we can delete the skip section and merge both of them as below based on the above?

## Change a maintenance schedule

A maintenance schedule can be updated or changed at any time. If the selected instance is going through an active maintenance cycle, the settings will be saved. If the selected instance is going through an active maintenance cycle, you can change the window start time as long as you do so before the beginning of your defined maintenance window, review the available options based on your specific circumstances.

- If you receive a pending notification for maintenance, and you need more time to finish your jobs or notify your team, you can change the window start time as long as you do so before the beginning of your defined maintenance window. This will shift your window forward in time within the cycle. Note that if you change the window to a start time before the actual present time, maintenance will be triggered immediately.
- You can manually trigger the maintenance by pausing and resuming (or scaling) your SQL Dedicated pool after the start of a cycle for which a "Pending" notification has been received. The weekend maintenance cycle starts on Saturday at 00:00 UTC; the midweek maintenance cycle starts Tuesday at 12:00 UTC.
- Although we do require a minimum window of 3 hours, note that maintenance usually takes less than 30 minutes to complete, but it may take longer in some cases. For example, if there are active transactions when the maintenance starts, they will be aborted and rolled back, which may cause delays in coming back online. To avoid this scenario, we recommend that you ensure that no long-running transactions are active during the start of your maintenance window.

Reply:
Possible Suggestion:  perhaps Sowmya can weigh in?

You can change your maintenance schedule at any time and the settings will be saved.  However, if you have already received a “Pending” maintenance notification for an upcoming maintenance operation, the changes may be applied without further notification messages being issued.

To ensure compliance with latest security requirements, we are unable to accommodate requests to skip or delay these updates.  But you may be able to adjust your maintenance window within the current cycle depending on your situation. Here are some options and notes to consider:

There are two “maintenance cycles” per week: the weekend maintenance cycle starts on Saturday at 00:00 UTC and ends Monday at 00:00 UTC; the midweek maintenance cycle starts Tuesday at 12:00 UTC and ends Thursday at 12:00 UTC..
If you have received a “Pending” notification for maintenance, you can change the window start time within the same cycle as long as you do it before the window starts. If you try to change the window to a time that has already passed, maintenance will start right away (you can’t “skip” a maintenance by changing the start time).
You can manually trigger the maintenance early by pausing and resuming (or scaling) your SQL Dedicated pool after the cycle with the “Pending” notification has started.
Although we do require a minimum window duration of 3 hours to be defined, maintenance usually takes less than 30 minutes. Sometimes it may take longer, especially if there are active transactions that need to be rolled back. To avoid this, make sure there are no active long-running transactions when the maintenance starts.

Reply:
Yeah this latest version from Jim, version sounds more tied to reality, I think? If yes, let's go with this update.